R/dashboard_modules/04-learner_characteristics.R

```diff
@@ -199,8 +200,8 @@ learner_characteristics_server <- function(id) {
     output$download_data <- downloadHandler(
       ## Set filename ---------------------------------------------------------
       filename = function(name) {
-        raw_name <- paste0(input$provider, "-", input$year, "-", "-provider_summary")
-        extension <- if (input$file_type == "CSV (Up to 11.61 MB)") {
+        raw_name <- paste0("learner_characteristics_provider_summary")
```

Comment:
Where it's a single string, you can just have that without needing paste0, so this would do exactly the same thing (as is a tiny bit cleaner to read):

```
raw_name <- "learner_characteristics_provider_summary"
```